packages/oh-my-posh/build.sh

```diff
@@ -0,0 +1,34 @@
+TERMUX_PKG_HOMEPAGE=https://ohmyposh.dev
+TERMUX_PKG_DESCRIPTION="A prompt theme engine for any shell."
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@termux"
```

Comment:
Would you consider setting yourself as the maintainer for the package?
Doesn't mean you gotta do everything, but it gives us someone as a reference in case packaging issues pop up in the future.

It's not a requirement, we just prefer new packages to have a maintainer set.

Reply:
I suppose I could do that. Although I just did the bare minimum to make this PR work haha. I relied on *GitHub* to build the repo to save disk space. Personally I use *Pacman* in *Alpine Linux*, to build packages that don't need to be updated often.

Reply:
No worries, if you don't feel like taking on the package you're by no means obligated to.

Reply:
I suppose I could still be a maintainer, if this doesn't mean I'm 100% responsible for the package.